accessible-places-frontend/src/App.tsx

```diff
@@ -4,19 +4,22 @@ import { LoginPage } from "./pages/LoginPage/LoginPage";
 import { UploadPage } from "./pages/UploadPage/UploadPage";
 
 import PrivateRoute from "./components/PrivateRoute/PrivateRoute";
+import { SideDrawerContextProvider } from "./Context/SideDrawerContext/SideDrawerContext";
 
 function App() {
   return (
-    <Routes>
-      <Route path="/login" element={<LoginPage />} />
-      <Route path="/" element={<div>Accessible Places - Frontend</div>} />
-      <Route path="/upload" element={<PrivateRoute />}>
-        <Route path="/upload" element={<UploadPage />} />
-      </Route>
-      <Route path="/map" element={<PrivateRoute />}>
-        <Route path="/map" element={<MapPage />} />
-      </Route>
-    </Routes>
+    <SideDrawerContextProvider>
```

Comment:
Does this work for you ? I had an error from 'react-router-dom' telling me that all children of <Router/> should be <Route/> elements.

I am asking because in main.tsx the App.tsx component is directly under the <Router/> wrapper. And so it might throw an error.

Reply:
I moved everything and now there is no error